github-changelog-generator.php

```diff
@@ -1,242 +1,113 @@
 <?php
 
+// "autoload" our classes
+if (!class_exists('ins0\GitHub\Repository')) {
+    require_once 'src/Repository.php';
```

Comment:
would suggest to add  ``__DIR__`` to avoid ``chdir`` conflicts and a possible bc break. to be hornest i don't like classes that have a autoload hack within the class itself. Whats your opinion on this?

- Ship the class "as it" - user should deal with the autoloading (eg. composer or user itself)
- Or this autoload